documentation/src/components/page.ts

```diff
@@ -30,7 +30,7 @@ export class Page extends LayoutElement {
     }
 
     public resetScroll() {
-        if (!this.contentElement) return;
+        if (!this.contentElement || !this.contentElement.scroll) return;
```

Comment:
I have seen this with the web component polyfill before. The `scroll` method is autocompleted in the console, but evaluates to `undefined` when you call it. This works around that. I am not going to investigate more deeply at this time.